src/components/UserInfoModal/UserInfoModal.jsx

```diff
@@ -72,57 +90,102 @@ const UserInfoModal = ({
     }
   };
 
+  const handleLeaveGroup = () => {
+    const dataEventToSend = {
+      chatId,
+    };
+    sendMessageOrEvent(dataEventToSend, URLs.leaveOutGroupChat);
+    setFilteredPrivateChats(prevRooms =>
+      prevRooms.filter(chat => chat.chat.id !== chatId)
+    );
+    setChatData(null);
+    handleClose();
+    closeConfirmation();
+  };
+
   return (
-    <ModalWindowStyled
-      aria-labelledby="transition-modal-title"
-      aria-describedby="transition-modal-description"
-      open={open}
-      onClose={handleClose}
-      closeAfterTransition
-      slots={{ backdrop: Backdrop }}
-      slotProps={{
-        backdrop: {
-          timeout: 500,
-        },
-      }}
-    >
-      <Fade in={open}>
-        <InfoModalStyled>
-          <ButtonClose onClick={handleClose}>
-            <CloseIcon />
-          </ButtonClose>
-          <UserContactInfoStyled>
-            {userAvatarUrl ? (
-              <ModalAvatar
-                src={userAvatarUrl || undefined}
-                alt={`${userName}'s avatar`}
-              />
+    <>
+      <ModalWindowStyled
+        aria-labelledby="transition-modal-title"
+        aria-describedby="transition-modal-description"
+        open={open}
+        onClose={handleClose}
+        closeAfterTransition
+        slots={{ backdrop: Backdrop }}
+        slotProps={{
+          backdrop: {
+            timeout: 500,
+          },
+        }}
+      >
+        <Fade in={open}>
+          <InfoModalStyled>
+            <ButtonClose onClick={handleClose}>
+              <CloseIcon />
+            </ButtonClose>
+            <UserContactInfoStyled>
+              {userAvatarUrl ? (
+                <ModalAvatar
+                  src={userAvatarUrl || undefined}
+                  alt={`${userName}'s avatar`}
+                />
+              ) : (
+                <LetterAvatarStyled>{firstLetterOfName}</LetterAvatarStyled>
```

Comment:
It's possible to do a little bit shorter `{userName[0].toUpperCase()}` and remove const firstLetterOfName.

Reply:
done